test/unit/specs/components/ActivityCollector/attachClickActivityCollector.spec.js

```diff
@@ -20,6 +20,7 @@ describe("ActivityCollector::attachClickActivityCollector", () => {
   let handleError;
   beforeEach(() => {
     config.clickCollectionEnabled = true;
+    config.onBeforeLinkCollect = undefined;
```

Comment:
I would recommend removing this line. The reasoning is that without this line, we can test the most common case of a customer providing a config with no `onBeforeLinkCollect` property at all. As you have it, we're testing what happens when a customer sets a property of `onBeforeLinkCollect` with a value of `undefined`, which wouldn't be common.